src/common/lang.js

```diff
@@ -34,14 +35,18 @@ export const load = () => {
 
     $('#select_language li:not(:first)').click(function click() {
         const newLang = $(this).attr('class');
-        if (document.getElementById('bot-landing').classList.contains('hidden') === false) {
-            remove('setDueDateForBanner');
-            render(<BotLanding />, document.getElementById('bot-landing'));
-            elements.map(elem => document.querySelector(elem).classList.add('hidden'));
-            document.getElementById('bot-landing').classList.remove('hidden');
-            document.getElementById('bot-main').classList.remove('hidden');
-            $('.barspinner').hide();
-            document.location.search = `l=${newLang}`;
+        if (document.getElementById('bot-landing') !== null && document.getElementById('bot-landing') !== undefined) {
+            if (document.getElementById('bot-landing').classList.contains('hidden') === false) {
```

Comment:
or maybe :
`if (document.getElementById('bot-landing') !== null && document.getElementById('bot-landing') !== undefined &&  document.getElementById('bot-landing').classList.contains('hidden') === false)`

and then emove two else and make it one